asynchronous_rabi_experiment/code.py

```diff
@@ -1,12 +1,25 @@
 """
-Adds one to a function
+Flips the state of the object
 """
-def add_one(x):
-    """
-    Add one to the input argument
-
-    :param x: The parameter to add
-    :return: The result after adding one
-    """
-    return x + 1
-
+class Flippable(object): 
+    
+    STATE_1 = "STATE_1"
+    STATE_2 = "STATE_2"
+    
+    def __init__(self, initial_state: str) -> None:
+        """
+        The initializer. This is the first method called by the class in order
+        to create an object. It behaves very similarly to the ```setUp`` method
+        that we defined in tests. However, instead of running before every
+        test, it runs every time we create a new object of this type.
+        """
+        self.state = initial_state
```

Comment:
What happens if I put in an initial state other that ``"STATE_1"`` or ``"STATE_2"``?

I think there are two ways to solve this problem

1. Check whether the string has the correct form in the initializer. If it does not, throw a
``ValueError``. For more value on Exceptions, go [here](http://www.pythonforbeginners.com/error-handling/exception-handling-in-python)

2. Instead of using ``STATE_1`` and ``STATE_2`` as strings, we can use an ``Enum`` in order to let this
type-check. We might still have to throw the exception, though. For more information on enums, click
[here](https://docs.python.org/3/library/enum.html).